### DISCUSSION TITLE
Como reactivar una task desactivada/borrada?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#65] [2. Questions and Answers] [answered]
Hola Quisiera confirmar lo siguiente:

Si quiero activar una task, debo:
1.- activar la task desde bd
2.- Activar el channel de la task

Son esos dos pasos o existe un tercero?

Existirá una forma futura cuando se quiera reactivar una task, me reactive el channel?

Gracias

**Comment by acedward** [comment, marked as answer]
Solo es posible modificar tareas a través de la API de Tasks y métodos previstos en los Bot Parametrizados.

Ejemplo a través de un CURL
```
curl -X PATCH "https://www.cotalker.com/api/v3/tasks/[group-id]/task/[task-id]" \
  -H "Authorization: Bearer [token]" \
  -H "accept: application/json" \
  -H "Content-Type: application/json"  \
  -d "{ isActive: true }"
```

**Reply by acedward** [reply]
En el sitio interno en:  `Software > Docs > API V2 Docs` puedes encontrar el link interno a la documentación (aun no lo publicamos)

**Reply by NicolasPLG** [reply]
gracias eddie, pero si l hice por bd no habría problemas?

**Reply by acedward** [reply]
Si, vas a tener problemas. No se pueden activar / deactivar a través de la base de datos. Hace un tiempo levantaron un ticket porque estaban haciendo esto, tasks tienen muchas dependencias.

**Reply by NicolasPLG** [reply]
OK @acedward entiendo, entonces usaré el curl que me indicas.

Gracias
